### PR TITLE
Consolidation of different git repository type into one unified instance

### PR DESCRIPTION
**Pull request by skoehnen**
As requested in #346

### CODE REVIEW

**Comment by lzehl** [comment]
@skoehnen thank you for taking this over. What about git-annex though? and I think we have to review the definitions of the other terms in this terminology (so that they describe the peculiarities of the repository type)

File: instances/fileRepositoryType/git.jsonld [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "preferredOntologyIdentifier": null,
+    "synonym": [
+        "Git repository",
+        "Git remote repository",
+        "remote Git repository",
+        "GitHub repository",
+        "GitLab repository"
+    ]
+  }
+
```
Comment (lzehl) [inline instances/fileRepositoryType/git.jsonld:21]: newline missing at the end of the jsonld

**Comment by skoehnen** [comment]
> @skoehnen thank you for taking this over. What about git-annex though? and I think we have to review the definitions of the other terms in this terminology (so that they describe the peculiarities of the repository type)

I would keep git-annex separate, while it share the same mechanics to store version-information, the handling of data is quite different.

**Comment by lzehl** [comment]
@skoehnen what I mean is could you please also create an instance for git-annex (since it is also a repository type) ?

**Comment by skoehnen** [comment]
> @skoehnen what I mean is could you please also create an instance for git-annex (since it is also a repository type) ?

@lzehl I will start a new PR for git-annex, can we merge this pull-request?

**Review by apdavison — APPROVED** [review]

**Review by lzehl — APPROVED** [review]

**Pull request merged**